src/js/views/forms/form/form_views.js

```diff
@@ -270,6 +270,18 @@ const ReadOnlyView = View.extend({
   `,
 });
 
+const RestrictedSubmitView = View.extend({
+  className: 'form__locked',
+  template: hbs`
+    <div class="form__locked-icon">
+      {{far "lock-keyhole"}}
+    </div>
+    <div class="form__locked-text">
+      {{ @intl.forms.form.formViews.restrictedSubmitView.permissionMessage }}
+    </div>
+  `,
+});
```

Comment:
Any thoughts on the naming I'm using? `RestrictedSubmitView`, `isSubmitRestricted`... I wasn't 100% what we want to name this stuff.

I didn't want to use any `lock` wording to avoid confusion with `action.isLocked()` (`locked_at` attribute). Which is a separate thing from this permission message we're showing here.

Reply:
If we stick with the current naming, I should change these class names to `form__restricted-submit` or something like that.